stl/inc/atomic

```diff
@@ -351,6 +413,36 @@ struct _Atomic_storage {
         return _Result;
     }
 
+#if _HAS_CXX20
+    void wait(const _Ty _Expected, const memory_order _Order = memory_order_seq_cst) const noexcept {
+        unsigned long long _Wait_context = 0;
+        const auto _Storage_ptr          = _STD addressof(_Storage);
+        const auto _Expected_ptr         = _STD addressof(_Expected);
+        for (;;) {
+            _Ty _Observed = load(_Order);
+            if (_CSTD memcmp(_STD addressof(_Observed), _Expected_ptr, sizeof(_Ty)) != 0) {
+                break;
+            }

```

Comment:
There should be a newline between these non-chained `if` statements, for clarity. Occurs below.